src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
+
   when (Done) {
     case Event(IssueReply, data) =>
+      isDone = true
```

Comment:
@dgtester As @cjllanwarne said that's true (-ish, there's an unlikely flaw in there) as long as one is using the `run` variant, but if one wants to use `server` it sits there and runs as many workflows as are submitted to it. 

Something which was brought up by our DevOps is that internally when we deploy a new version that a SIGINT ends up being sent but we *don't* want workflows to be aborted there. 

To me this seems like behavior one would only want in a single workflow mode.

I wasn't kidding last night when I said my comments were based purely on a quick scan and that continues to be true, so I really haven't thought too much about what should/could happen, what's possible, etc.